WebRoot/js/zimbraMail/share/model/ZmFolder.js

```diff
@@ -171,7 +166,6 @@ ZmFolder.TCON_CODE[ZmFolder.ID_OTHER]			= "o";
 
 // folders that look like mail folders that we don't want to show
 ZmFolder.HIDE_ID = {};
-ZmFolder.HIDE_ID[ZmOrganizer.ID_CHATS]				= true;
```

Comment:
This caused existing Chat folder to become visible (unhidden) again:
https://forums.zimbra.org/viewtopic.php?p=312138#p312138
Please keep it hidden (or add a script to remove it from all mailboxes).